Assets/Art/leaderheads/Afonso/Afonso_CIV4LeaderHeadInfos.xml

```diff
@@ -1,571 +0,0 @@
-<?xml version="1.0"?>
-<!-- Sid Meier's Civilization 4 Beyond the Sword -->
-
-<!-- Modified by the Civ Gold Team -->
-
-<!-- -->
-<Civ4LeaderHeadInfos xmlns="x-schema:Afonso_CIV4CivilizationsSchema.xml">
-<LeaderHeadInfo>

```

Comment:
I think Afonso currently uses a copy of some other LH's personality, maybe he should use this one instead?